modin/experimental/core/execution/native/implementations/hdk_on_native/hdk_worker.py

```diff
@@ -143,7 +142,7 @@ def compute_fragment_size(cls, table):
         fragment_size = HdkFragmentSize.get()
         if fragment_size is None:
             if cls._preferred_device == "CPU":
-                cpu_count = os.cpu_count()
+                cpu_count = CpuCount.get()
```

Comment:
maybe it makes more sense to control this via `NPartitions` variable, what do you think?

Reply:
The number of HDK threads is limited by CpuCount - https://github.com/modin-project/modin/blob/master/modin/pandas/__init__.py#L145. AFAIK, MODIN_CPUS is used in docker images to specify the available number of CPUs.

Reply:
> The number of HDK threads is limited by CpuCount

Right, the number of HDK's threads is limited by CpuCount, but here, the `cpu_count` variable explicitly defines the number of fragments (partitions), so my question is, whether it makes sense to use `NPartitions` here to specify the number of fragments and CpuCount to separately specify the number of HDK's threads?

Reply:
We already have the MODIN_HDK_FRAGMENT_SIZE variable for this purpose. If it's not specified, we use a heuristic, based on the table size and the number of cpus.

NPartitions could be confusing, I think, because it's known, that HDK uses a single partition.